test/e2e/framework/util.go

```diff
@@ -4438,6 +4438,8 @@ const (
 	IPv4PingCommand PingCommand = "ping"
 	// IPv6PingCommand is a ping command for IPv6.
 	IPv6PingCommand PingCommand = "ping6"
+	// IPv4NcCommand is a nc command for IPv4
+	IPv4NcCommand PingCommand = "nc"
```

Comment:
`nc` is not a ping command, does it belong in this type?

Reply:
Separated it out.